usaspending_api/idvs/v2/views/awards.py

```diff
@@ -199,8 +206,26 @@ def _business_logic(request_data: dict) -> list:
             limit=Literal(request_data["limit"] + 1),
             offset=Literal((request_data["page"] - 1) * request_data["limit"]),
         )
+        results = execute_sql_to_ordered_dictionary(sql)
 
-        return execute_sql_to_ordered_dictionary(sql)
+        funding_agency_slugs = generate_agency_slugs_for_agency_list(
```

Comment:
No need to build a dictionary of "funding" and "awarding" Agency slugs. For example, DoD as an Awarding Agency will have the same slug as a Funding Agency. And if DoD has a Submission then they will return a slug for both Awarding and Funding. This is just adding an extra DB call.

Reply:
I combined the list of Toptier IDs then made one call to `generate_agency_slugs_for_agency_list(toptier_ids)`